### PR TITLE
Use standard generics for TextSize::of

### PR DESCRIPTION
**Pull request by CAD97**
There is no specific reason to use APIT here, so prefer the form that allows more control for the user, in the form of the turbofish.

### CODE REVIEW

File: src/size.rs [hunk 1]
```diff
@@ -42,7 +42,7 @@ impl fmt::Debug for TextSize {
 impl TextSize {
     /// The text size of some text-like object.
     #[inline]
-    pub fn of(text: impl TextSized) -> TextSize {
+    pub fn of<Text: TextSized>(text: Text) -> TextSize {
```
Comment (matklad) [inline src/size.rs]:
```suggestion
    pub fn of<T: TextSized>(text: T) -> TextSize {
```

**Comment by matklad** [comment]
yeah, I've meant to do this myself for exactly the same reason,

bors d+

(needs a rebase)

**Comment by CAD97** [comment]
Rebased.

bors: r+

**Comment by CAD97** [comment]
Whoops, bad rebase.

bors: retry

**Pull request merged**